client/web/compose/src/components/PageBlocks/Configurator.vue

```diff
@@ -96,9 +96,24 @@
 
         <b-form-group
           v-if="block.options.refreshRate !== undefined"
```

Comment:
Probably best to change the condition to the options.refresh property, since its more important. Same for anywhere else that you use this condition